staging/src/k8s.io/client-go/examples/create-update-delete-deployment/main.go

```diff
@@ -138,9 +139,7 @@ func main() {
 			panic(fmt.Errorf("Failed to get latest version of Deployment: %v", getErr))
 		}
 
-		result.Spec.RollbackTo = &appsv1beta1.RollbackConfig{
-			Revision: 0, // can be specific revision number, or 0 for last revision
-		}
+		result.Annotations[appsv1.DeprecatedRollbackTo] = strconv.FormatInt(0, 10) // can be specific revision number, or 0 for last revision
```

Comment:
doesn't seem like we should move to something deprecated in an example...

Reply:
removed it